Update keyboard-shortcuts.md (#33)
FreeTubeApp/FreeTube-Docs: usage/keyboard-shortcuts.md

| Alt + D or Command/Ctrl + L                     | Focus on the search bar                                                                                                                                                                              |
| Play or Pause                                   | Play / Pause video                                                                                                                                                                                   |
| Previous or Next                                | Skips to the Previous / Next video in a playlist                                                                                                                                                     |
| , or .                                          | While the video is paused, skip to the previous / next frame.                                                                                                                                        |

> | SHIFT + Enter or SHIFT + Click on Search button | Search in a new instance of FreeTube                                                                                                                                         |

Reviewer: Search/Browse Video, Playlist etc. in a new window

Author: https://github.com/FreeTubeApp/FreeTube-Docs/pull/33/commits/59eea202e3039fe0db180353a16982c648c14ab7

> | SHIFT + Enter or SHIFT + Click on Search button | After user puts in search query use on of the following shortcuts to perform the search in a new instance                                                                                            |

Reviewer 2: Could be a bit more concise
```suggestion
| SHIFT + Enter or SHIFT + Click on Search button | Search in a new instance of FreeTube                                                                                                                                         |
```

Reviewer 2: Let me know your thoughts on this change

Author: Yes i completely agree. I didnt know how to properly word it.
